### ISSUE TITLE
Add shutdown host possibility for Sunshine hosts

### ISSUE THREAD

**Issue by apoklyps3** [moonlight-stream/moonlight-qt#934]
Since sunshine isn't bound by nvidia gamestream limitations it would be cool if shutdown & restart host would be added to the right-click menu on a host computer.
Wouldn't hurt if we got key combos for these actions too.
I asked on the sunshine github and they claim shutdown and restart host is already supported on their server, but they have to be implemented in moonlight too. Is that correct?

**Comment by ivanmara555** [comment]
Good idea to add logout option to moonlight, because some times game freeze and does not respond to key presses on the keyboard ... but system still working. Usually in this situation i connect to RDP as Administrator and shutdown game process manually. Logout in moonlight would be more comfortable.

**Comment by jorisdejosselin** [comment]
As a workaround I use an application in Sunshine with the detached command `cmd /C "C:\Windows\system32\shutdown.exe /s /t 5"` but it would be nice to have it as an option in the host menu.

**Comment by IvoNexus** [comment]
To advance the @jorisdejosselin idea, place this `cmd /C "C:\Windows\system32\shutdown.exe /s /t 300"` in "Command Preparations > Undo Command" and this `cmd /C "C:\Windows\system32\shutdown.exe /a >nul 2>&1"` in Detached Commands for "Steam Big Picture", the idea is to set shutdown period of 5 minutes, however if you decide to reconnect within that time system will cancel the scheduled shutdown procedure. Feel free to change the time to your needs, this was inspired by the Geforce Now cloud behavior and playing on workplace 🤙
